cluster/mesos/docker/util.sh

```diff
@@ -317,7 +317,8 @@ function kube-down {
 }
 
 function test-setup {
-  echo "TODO: test-setup" 1>&2
+  echo "test-setup" 1>&2
```

Comment:
It has `echo "test-teardown" 1>&2` in `test-teardown`, but the juju one doesn't.